### DISCUSSION TITLE
URL, images, et customfile

### DISCUSSION THREAD

**Discussion by jodionrq** [MTESSDev/FRW#22] [Annonces]
Bonjour, 

1. Lors que nous plaçons le bouton de type « customfile » dans le but de permettre l’utilisateur de téléverser un fichier, est que les fichiers envoyés seront déposés sur le serveur du MESS? 

2. Est-ce possible d’ajouter un lien URL dans du texte, c.-à-d. que l’utilisateur puisse cliquer sur du texte qui mène à une page Web ou un PDF sur le site de Retraite Québec?

3. Pour ajouter une image avec source, est-ce que l’image doit être déposée sur un serveur, ou est-ce possible de prendre une image dans un répertoire « local » le temps de faire nos essais?

Merci!

**Comment by j0ber** [comment]
S.V.P une question un sujet :)  La question en est fait fait 3.

**Comment by j0ber** [comment]
**Réponse 1**
Oui nous conservons les fichiers un certain temps sur nos serveurs (tout est crypté). Vous pouvez utiliser le service "[telecharger](https://github.com/MTESSDev/FRW/tree/main/Swagger#apiv1documenttelechargernomfichierprotege)" afin d'obtenir les fichiers.

**Réponse 2**
Il faut savoir de quel texte vous parlez, mais oui c'est possible. Il faudrait avoir votre cas de figure particulier. S.V.P. créer une question avec du détail sur le besoin et nous y répondrons.

**Réponse 3**
Pour le moment il faut convertir l'image en base 64 (utiliser un outil en ligne) et l'intégrer dans votre fichier de config. Voir l'exemple pour une [image en base 64](https://formulaires.it.mtess.gouv.qc.ca/Form/1/P700U/0/N#p=50).

Nous allons également ajouter la possibilité d'utiliser une image qui est hébergée sur n'importe quel site ayant comme domaine "gouv.qc.ca". (à venir bientôt)

**Comment by jodionrq** [comment]
Bonjour,

Merci beaucoup d'avoir pris le temps de répondre à mes questions. C'est noté pour mes prochaines demandes.
Les réponses pour l'image et le bouton "télécharger" sont parfaites. Je vais créer une autre demande pour le lien URL avec plus de précisions.
Bonne journée!
